### PR TITLE
errors in the doc

### PR DESCRIPTION
**Pull request by kineticz**

### CODE REVIEW

File: src/libcore/iter/iterator.rs [hunk 1]
```diff
@@ -214,7 +214,7 @@ pub trait Iterator {
     /// Like most indexing operations, the count starts from zero, so `nth(0)`
     /// returns the first value, `nth(1)` the second, and so on.
     ///
-    /// `nth()` will return `None` if `n` is larger than the length of the
+    /// `nth()` will return `None` if `n` >= the length of the
```
Comment (steveklabnik) [inline src/libcore/iter/iterator.rs]: I would prefer to stick with the words here, "is greater than or equal to the length"

**Comment by steveklabnik** [comment]
Thanks for this! One small nit and then r=me

**Comment by kindlychung** [comment]
r? @steveklabnik  

Done.

**Comment by steveklabnik** [comment]
@bors: r+ rollup

Thank you!

**Pull request merged**